docs/source/conf.py

```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
```

Comment:
why removing this line?  can't we still have files using latin 1 or other encoding?